spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/AppStatusResource.java

```diff
@@ -44,6 +44,12 @@ public AppStatusResource(String deploymentId, String state) {
 		this.state = state;
 	}
 
+	public String getName() {
+		AppInstanceStatusResource instance = this.instances.iterator().next();
+		return (instance != null && instance.getAttributes() != null) ?
+				instance.getAttributes().get("skipper.application.name") : "no-instances";
```

Comment:
I think we need to find a better place to fit these attribute names as constants (the skipper.application.name and skipper.release.version). Currently, they are declared in the controller package but not in scope of the REST resources.

Reply:
I will factor those at least as constants. 
But i agree they should be pulled in some parent (`spring-cloud-dataflow-core` perhaps) that is shared with the controllers as well.